data/json/mutations/mutations.json

```diff
@@ -9166,6 +9166,16 @@
     "purifiable": false,
     "profession": true
   },
+  {
+    "type": "mutation",
+    "id": "PROF_FBI",
+    "name": { "str": "Special Agent" },
+    "points": 0,
+    "description": "You are a member of the Federal Bureau of Investigation, also known as the FBI.  You have jurisdiction throughout the remaining 49 States, Indian Country, and any other territories of the USA.  Whether that means anything now is another question.",
+    "valid": false,
+    "purifiable": false,
+    "profession": true
+  },
```

Comment:
Either hook this up to the same C++ code the other police 'mutations' are connected to, or remove this. Currently this is useless JSON.

Reply:
How would I go about doing that?